### PR TITLE
Added "List tiers" operation

### PR DESCRIPTION
**Pull request by ArthyVenk**

### CODE REVIEW

**Review by umavanth21 — COMMENTED** [review]
are we not using clientid and clientsecret for this operation?

**Comment by ArthyVenk** [comment]
> 

No uma. We will get only the access token. We are not generating access token here in this operation

**Comment by umavanth21** [comment]
> > 
> 
> No uma. We will get only the access token. We are not generating access token here in this operation

ok got it Arthy

**Review by umavanth21 — APPROVED** [review]

File: content/io/cloudslang/google/databases/list_tiers.sl [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+#!                        Default value: '..JAVA_HOME/java/lib/security/cacerts'
+#!                        Format: Java KeyStore (JKS)
+#!                        Optional
+#! @input trust_password: The password associated with the trust_keystore file. If trust_all_roots is false
+#!                        and trust_keystore is empty, trust_password default will be supplied.
+#!                        Optional
+#!
+#! @output return_result: This will contain the response entity.
+#! @output status_code: 200 if request completed successfully, others in case something went wrong.
+#! @output instances_json: A JSON list containing the Instances information.
```
Comment (patmante) [inline content/io/cloudslang/google/databases/list_tiers.sl]: change the output name and description

File: content/io/cloudslang/google/databases/list_tiers.sl [hunk 2]
```diff
@@ -0,0 +1,160 @@
...
+          - status_code
+        navigate:
+          - SUCCESS: set_success_message
+          - FAILURE: on_failure
+    - set_success_message:
+        worker_group: '${worker_group}'
+        do:
+          io.cloudslang.base.utils.do_nothing:
+            - message: The list of all available machine types (tiers) has been successfully retrieved.
+            - instances_json: '${return_result}'
```
Comment (patmante) [inline content/io/cloudslang/google/databases/list_tiers.sl]: here also update the output name

**Review by patmante — CHANGES_REQUESTED** [review]

File: content/io/cloudslang/google/databases/list_tiers.sl (hunk 2 above)
Comment (ArthyVenk) [reply content/io/cloudslang/google/databases/list_tiers.sl]: Updated

**Review by patmante — APPROVED** [review]

**Pull request merged**